### PR TITLE
Move the error flash insertion to a javascript function

### PR DESCRIPTION
**Pull request by whaught**
## Proposed Changes

* Move the error flash insertion to a javascript function

### CODE REVIEW

File: cmd/server/assets/header.html [hunk 1]
```diff
@@ -208,5 +209,28 @@ <h6 class="dropdown-header">Actions</h6>
...
     })
   });
+
+  var $main = $('main');
+
+  function clearExistingAlerts() {
+    $main.find('div.alert').remove();
+  }
+
+  function alert(message, errLevel) {
```
Comment (sethvargo) [inline cmd/server/assets/header.html:219]: `alert` is a javascript built-in technically it's window.alert, but it also exists in the default namespace. Maybe name this `flash`?

Comment (whaught) [reply cmd/server/assets/header.html]: done

**Pull request merged**